rtengine/rawimagesource.h

```diff
@@ -245,7 +245,7 @@ class RawImageSource : public ImageSource
     inline  void interpolate_row_rb     (float* ar, float* ab, float* pg, float* cg, float* ng, int i);
     inline  void interpolate_row_rb_mul_pp (float* ar, float* ab, float* pg, float* cg, float* ng, int i, float r_mul, float g_mul, float b_mul, int x1, int width, int skip);
 
-    void CA_correct_RT  (const bool autoCA, const double cared, const double cablue, const double caautostrength, array2D<float> &rawData);
+    float* CA_correct_RT  (const bool autoCA, const double cared, const double cablue, const double caautostrength, array2D<float> &rawData, double *fitParamsTransfer = nullptr, bool fitParamsIn = false, bool fitParamsOut = false, float * buffer = nullptr, bool freeBuffer = true);
```

Comment:
`CA_correct_RT` is only used in line 2020 et seq. of `rtengine/rawimagesource.cc`, so I don't think there's a need for all those defaults. It's harder to review and debug when default arguments are involved. Maybe consider approriate forwarding overloads.